app/config_files/templates.json

```diff
@@ -1,274 +1,347 @@
-            "",
-            "Thanks,",
-            "Notify.gov team"
-        ]
-    },
-    {
-        "id": "522b6657-5ca5-4368-a294-6b527703bd0b",
-        "name": "MOU Signed On Behalf Of Receipt - On Behalf Of",
-        "type": "email",
-        "subject": "((org_name)) has accepted the Notify.gov data sharing and financial agreement",
-        "content": [
-            "Hi ((on_behalf_of_name)),",
-            "",
-            "((signed_by_name)) has accepted the Notify.gov data sharing and financial agreement on your behalf, for ((org_name)).",
-            "",
-            "Notify.gov lets teams in the public sector send text messages and emails. It’s built and run by a team in the TTS Public Benefits Studio (part of GSA).",
-            "",
-            "If you need another copy of the agreement you can download it here: ((mou_link))",
-            "",
-            "",
-            "Thanks,",
-            "Notify.gov team"
-        ]
-    },
-    {
-        "id": "d0e66c4c-0c50-43f0-94f5-f85b613202d4",
-        "name": "MOU Signed Notify Team Alert",
-        "type": "email",
-        "subject": "Someone signed an MOU for an org on Notify",
-        "content": [
-            "What’s up Notifiers,",
-            "",
-            "((signed_by_name)) just accepted the data sharing and financial agreement for ((org_name)).",
-            "",
-            "See how ((org_name)) is using Notify here: ((org_dashboard_link))"
-        ]
-    },
-    {
-        "id": "11fad854-fd38-4a7c-bd17-805fb13dfc12",
-        "name": "Notify daily letter volumes",
-        "type": "email",
-        "subject": "Notify letter volume for ((date)): ((total_volume)) letters, ((total_sheets)) sheets",
-        "content": [
-            "((total_volume)) letters (((total_sheets)) sheets) sent via Notify are coming in today’s batch. These include: ",
-            "",
-            "((first_class_volume)) first class letters (((first_class_sheets)) sheets).",
-            "((second_class_volume)) second class letters (((second_class_sheets)) sheets).",
-            "((international_volume)) international letters (((international_sheets)) sheets).",
-            "",
-            "Thanks",
-            "",
-            "Notify.gov team"
-        ]
-    }
+  {
+    "id": "42a23d19-504e-49bb-a95e-4976baff4757",
+    "name": "Example text message template",
+    "type": "sms",
+    "subject": "",
+    "content": [
+      "Hi, I’m trying out Notify.gov. Today is ((day of week)) and my favorite color is ((color))."
+    ]
+  },
+  {
+    "id": "4f46df42-f795-4cc4-83bb-65ca312f49cc",
+    "name": "Notify invitation email",
+    "type": "email",
+    "subject": "((user_name)) has invited you to collaborate on ((service_name)) on Notify.gov",
+    "content": [
+      "((user_name)) has invited you to collaborate on ((service_name)) on Notify.gov.",
+      "",
+      "",
+      "Notify.gov makes it easy to keep people updated by helping you send text messages.",
```

Comment:
I removed "and emails"

Reply:
Strange that the entire JSON got flagged for the diff despite the one line text change!  Thanks for calling out the difference, @heyitsmebev!